When i am trying too install This :  npm install github:alechkos/whatsapp-web.js#add-polls

npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/alechkos/whatsapp-web.js.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-08-23T09_53_00_971Z-debug.log
[root@localhost node]# npm i github:alechkos/whatsapp-web.js#add-polls
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/alechkos/whatsapp-web.js.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-08-23T09_54_47_928Z-debug.log

--- a/src/Client.js
+++ b/src/Client.js
@@ -285,13 +285,13 @@ class Client extends EventEmitter {
              */
             window.compareWwebVersions = (lOperand, operator, rOperand) => {
                 if (!['>', '>=', '<', '<=', '='].includes(operator)) {
-                    throw class _ extends Error {
+                    throw new class _ extends Error {
                         constructor(m) { super(m); this.name = 'CompareWwebVersionsError'; }
                     }('Invalid comparison operator is provided');
 
                 }
                 if (typeof lOperand !== 'string' || typeof rOperand !== 'string') {
-                    throw class _ extends Error {
+                    throw new class _ extends Error {
                         constructor(m) { super(m); this.name = 'CompareWwebVersionsError'; }
                     }('A non-string WWeb version type is provided');
                 }

--- a/src/util/Injected.js
+++ b/src/util/Injected.js
@@ -80,11 +80,8 @@ exports.ExposeStore = (moduleRaidStr) => {
         };
     }
 
-    if (window.mR.findModule('ChatCollection')[0] && window.mR.findModule('ChatCollection')[0].ChatCollection) {
-        if (typeof window.mR.findModule('ChatCollection')[0].ChatCollection.findImpl === 'undefined' && typeof window.mR.findModule('ChatCollection')[0].ChatCollection._find != 'undefined') {
-            window.mR.findModule('ChatCollection')[0].ChatCollection.findImpl = window.mR.findModule('ChatCollection')[0].ChatCollection._find;
-        }
-    }
+    // eslint-disable-next-line no-undef
+    if ((m = window.mR.findModule('ChatCollection')[0]) && m.ChatCollection && typeof m.ChatCollection.findImpl === 'undefined' && typeof m.ChatCollection._find !== 'undefined') m.ChatCollection.findImpl = m.ChatCollection._find;
 
     // TODO remove these once everybody has been updated to WWebJS with legacy sessions removed
     const _linkPreview = window.mR.findModule('queryLinkPreview');

